src/components/album/album-comments/comment.component.js

```diff
@@ -0,0 +1,35 @@
+import template from './comment.html';
+
+export const CommentComponent = {
+	template,
+	bindings: {
+		albumId: '<'
+	},
+	controller: class CommentComponent{
+		constructor(AlbumService){
+			'ngInject';
+			this.albumService = AlbumService;
+			this.comments;
+			this.enableNewComment = false;
+		}
+
+		$onInit(){
+			this.getComments();
+		}
+
+		toggleNewComment(){
+			this.enableNewComment = !this.enableNewComment;
+		}
+
+		getComments(){
+			this.albumService.getCommentsByAlbumId(this.albumId).then(response => {
+				this.comments = response.data;
+			});
+		}
+
+		addComment({comment}){
+			comment.albumId = this.albumId;
+			this.albumService.postNewComment(comment).then(response => {this.getComments()});
```

Comment:
So you're saying that the main container (album.component) should handle that operation